blueocean-dashboard/src/main/js/redux/actions.js

```diff
@@ -57,7 +57,23 @@ export const actionHandlers = {
         return state.set('currentRuns', null);
     },
     [ACTION_TYPES.SET_CURRENT_RUN_DATA](state, { payload }): State {
-        return state.set('currentRuns', payload);
+        return state.set('currentRuns', payload.map((run) => {
+            if (run._class === 'io.jenkins.blueocean.service.embedded.rest.QueueItemImpl') {
+                return {
+                    id: String(run.expectedBuildNumber),
```

Comment:
`id: String(run.expectedBuildNumber)` is that right? Is it missing a `new` operator? Or maybe just do `run.expectedBuildNumber.toString()`?